egs/vctk/TTS/prepare.sh

```diff
@@ -0,0 +1,132 @@
+  fi
+fi
+
+if [ $stage -le 2 ] && [ $stop_stage -ge 2 ]; then
+  log "Stage 2: Compute spectrogram for VCTK"
+  mkdir -p data/spectrogram
+  if [ ! -e data/spectrogram/.vctk.done ]; then
+    ./local/compute_spectrogram_vctk.py
+    touch data/spectrogram/.vctk.done
+  fi
+
+  if [ ! -e data/spectrogram/.vctk-validated.done ]; then
+    log "Validating data/fbank for VCTK"
+    ./local/validate_manifest.py \
+      data/spectrogram/vctk_cuts_all.jsonl.gz
+    touch data/spectrogram/.vctk-validated.done
+  fi
+fi
+
+if [ $stage -le 3 ] && [ $stop_stage -ge 3 ]; then
+  log "Stage 3: Prepare phoneme tokens for VCTK"
+  if [ ! -e data/spectrogram/.vctk_with_token.done ]; then
+    ./local/prepare_tokens_vctk.py
+    mv data/spectrogram/vctk_cuts_with_tokens_all.jsonl.gz \
+      data/spectrogram/vctk_cuts_all.jsonl.gz
+    touch data/spectrogram/.vctk_with_token.done
+  fi
+fi
+
+if [ $stage -le 4 ] && [ $stop_stage -ge 4 ]; then
+  log "Stage 4: Split the VCTK cuts into train, valid and test sets"
+  if [ ! -e data/spectrogram/.vctk_split.done ]; then
+    lhotse subset --last 600 \
+      data/spectrogram/vctk_cuts_all.jsonl.gz \
+      data/spectrogram/vctk_cuts_validtest.jsonl.gz
+    lhotse subset --first 100 \
+      data/spectrogram/vctk_cuts_validtest.jsonl.gz \
+      data/spectrogram/vctk_cuts_valid.jsonl.gz
+    lhotse subset --last 500 \
+      data/spectrogram/vctk_cuts_validtest.jsonl.gz \
+      data/spectrogram/vctk_cuts_test.jsonl.gz
+
+    rm data/spectrogram/vctk_cuts_validtest.jsonl.gz
+
+    n=$(( $(gunzip -c data/spectrogram/vctk_cuts_all.jsonl.gz | wc -l) - 600 ))
+    lhotse subset --first $n  \
+      data/spectrogram/vctk_cuts_all.jsonl.gz \
+      data/spectrogram/vctk_cuts_train.jsonl.gz
+      touch data/spectrogram/.vctk_split.done
+  fi
+fi
+
+if [ $stage -le 5 ] && [ $stop_stage -ge 5 ]; then
+  log "Stage 5: Generate token file"
+  # We assume you have installed g2p_en and espnet_tts_frontend.
+  # If not, please install them with:
+  #   - g2p_en: `pip install g2p_en`, refer to https://github.com/Kyubyong/g2p
+  #   - espnet_tts_frontend, `pip install espnet_tts_frontend`, refer to https://github.com/espnet/espnet_tts_frontend/
+  if [ ! -e data/tokens.txt ]; then
+    ./local/prepare_token_file.py \
+      --manifest-file data/spectrogram/vctk_cuts_train.jsonl.gz \
+      --tokens data/tokens.txt
+  fi
+fi
+
+if [ $stage -le 6 ] && [ $stop_stage -ge 6 ]; then
+  log "Stage 6: Generate speakers file"
+  if [ ! -e data/speakers.txt ]; then
+    gunzip -c data/manifests/vctk_supervisions_all.jsonl.gz \
+      | jq '.speaker' | sed 's/"//g' \
+      | sort | uniq > data/speakers.txt
+  fi
+fi
+
+if [ $stage -le 7 ] && [ $stop_stage -ge 7 ]; then
+  log "Stage 7: build monotonic_align lib"
+  if [ ! -d vits/monotonic_align/build ]; then
+    cd vits/monotonic_align
+    python setup.py build_ext --inplace
+    cd ../../
+  else 
+    log "monotonic_align lib already built"
+  fi
+fi
```

Comment:
Could you add this part to the script `prepare.sh` in `ljspeech` recipe, and also correspondingly modify the doc?